README.md

```diff
@@ -83,6 +83,11 @@ Now to use `snarkvm`, in your terminal, run:
 snarkvm
 ```
 
+## 3. Contributing
+
+PRs and issues are welcome. Please see the [contribution guide](./CONTRIBUTING.md) for more
+information about how to create a successful PR.
+
 ## 3. Usage Guide
```

Comment:
Nit: the numbering and ToC needs to be updated

Reply:
It looks like GitHub has recently added a feature to auto-generate README ToCs. Should we try that?

Reply:
And/or perhaps we drop the `3. Usage Guide` stub for now?

Reply:
For now just updated it without the automated GitHub ToC